src/test/java/ui/ProductAddTest.java

```diff
@@ -0,0 +1,35 @@
+package ui;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import uiPages.LoginPage;
+import uiPages.ProductPage;
+
+import static helpers.Utilities.getPropertyFromAppProp;
+
+public class ProductAddTest extends BaseTest {
+
+    private ProductPage productPage = null;
+
+    private String specificProduct;
+
+    public ProductAddTest() {
+        this.specificProduct = getPropertyFromAppProp("product1");
+    }
+
+    @BeforeMethod
+    void beforeMethod() {
+        this.productPage = new ProductPage(getDriver());
+    }
+
+    @Test
+    public void addProductToCart() {
+        this.productPage.open(specificProduct);
+        this.productPage.verify();
+        this.productPage.attributeSelector();
+        this.productPage.addToCart();
+        Assert.assertTrue(this.productPage.isAddedMessage(), "The product was NOT added to the cart");
```

Comment:
I noticed the added to cart message contains the product name as well. Validate that the product name is correct too.

Reply:
I will address this in a future branch.